addons/sale_timesheet/data/sale_service_demo.xml

```diff
@@ -31,6 +31,7 @@
             <field name="name">Internal training</field>
             <field name="user_id" ref="base.user_admin"/>
             <field name="project_id" ref="project.project_project_1"/>
+            <field name="create_date" eval="DateTime.now() - relativedelta(months=5)"/>
```

Comment:
I think this change is useless since we do not create any tracking values for this task 🙂

Reply:
@xavierbol ,
Yes, we does not have any tracking values but It has impact on graph as by default task will be in the `new` state and based on create date it will increase the new state values on the graph.
like: https://drive.google.com/file/d/1A7CfKyRSgsqB9tcaKj4vO_UufCRGJ90G/view?usp=sharing

Thanks

Reply:
Ok great, thanks for the explanation, I saw with AJU and it is ok as you have done. 🙂